frontend/src/features/StartLesson/lessonAPI.ts

```diff
@@ -1,23 +1,8 @@
 const axios = require("axios").default;
 
-export const fetchQuizDetail = async (quizId: number) => {
-  try {
-    const response = await axios.get(
-      `http://127.0.0.1:8000/api/quizzes/${quizId}`
-    );
-    return response.data;
-  } catch (error) {
-    console.error(error);
-  }
+export const fetchQuizDetail = (quizId: number) => {
+  return axios.get(`http://127.0.0.1:8000/api/quizzes/${quizId}`);
 };
-
-export const fetchQuestions = async (quizId: number) => {
-  try {
-    const response = await axios.get(
-      `http://127.0.0.1:8000/api/quizzes/${quizId}/questions`
-    );
-    return response.data;
-  } catch (error) {
-    console.error(error);
-  }
+export const fetchQuestions = (quizId: number) => {
+  return axios.get(`http://127.0.0.1:8000/api/quizzes/${quizId}`);
```

Comment:
you missed `/questions`

Also I encourage you to double check your code/implementation again before you push to avoid bugs being merged in master branch.